app/javascript/components/components/Header.test.js

```diff
@@ -9,7 +9,7 @@ describe('When Header renders', () => {
   it('displays a Header', () => {
     const header = shallow(<Header />);
     const headerLink = header.find('NavLink');
-    expect(headerLink.length).toEqual(4);
+    expect(headerLink.length).toEqual(1);
```

Comment:
Is there any text provided in the NavLink? This would be a good opportunity to assert on navigation or props passed into NavLink

Reply:
Also this looks like there's possibly more than one NavLink because you have more than one link in the header. Try to differentiate between the NavLinks by using ids or classes or different selectors and then assert on the different routes